cmd/docker/main.go

```diff
@@ -176,7 +176,27 @@ func startServerWithOptions(cli *CliOptions) *apiserver.Server {
 		Version: "1.22", //dockerversion.Version,
 	}
 
-	tlsConfig := tlsconfig.ServerDefault
+	tlsConfig := func(c *tls.Config) *tls.Config {
```

Comment:
cmd/docker/main.go:179: assignment copies lock value to tlsConfig: crypto/tls.Config contains sync.Once contains sync.Mutex